airlock/views/workspace.py

```diff
@@ -93,6 +93,17 @@ def workspace_view(request, workspace_name: str, path: str = ""):
             workspace=workspace.name, exclude_readonly=True, size=20
         )
 
+    if path_item.is_directory() or path not in workspace.manifest["outputs"]:
+        code_url = None
+    else:
+        code_url = reverse(
+            "code_view",
+            kwargs={
+                "workspace_name": workspace.name,
+                "commit": workspace.get_manifest_for_file(path)["commit"],
```

Comment:
Is it worth handling the case where the "commit" key isn't found?

Reply:
Can do, but will there ever be one? `code_url` will be `None` if the path isn't in `manifest["outputs"]`, shouldn't every output file have a commit key?

Reply:
It should, but we don't currently validate the manifest.json beyond parsing it as valid json, its possible a format change or an error might

I'd rather just break the code link than the whole page?

Reply:
I've changed it to a .get() so the page doesn't break on a KeyError, but I've also updated the code view so if the repo/commit doesn't exist it gives you an error message instead of just 404ing when you click the code button